subprojects/docs/build.gradle

```
@@ -557,3 +558,13 @@ class GradleInstallationForTestEnvironmentProvider implements CommandLineArgumen
          "-DintegTest.gradleUserHomeDir=${repoRoot(project).dir("intTestHomeDir/$distributionName").asFile}"]
     }
 }
+
+tasks.withType(CheckLinks) {
```

```suggestion
tasks.withType(CheckLinks).configureEach {
```

I am pretty sure we need `configureEach` here to not realize all the tasks.

---

Done.